From 3d4d3cda19751759a4c2aa7a43c8e131bde863dc Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:10:59 -0400
Subject: [ethernet] i40evf: don't guess device name

Message-id: <1409663591-32531-17-git-send-email-sassmann@redhat.com>
Patchwork-id: 90493
O-Subject: [RHEL7.1 PATCH 016/148] i40evf: don't guess device name
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Mitch Williams <mitch.a.williams@intel.com>

We don't need to set an interface name here; the net core will do
that, and then it will get renamed by udev anyway.

Change-ID: I839a17837d19bedd1f490bff32ac5b85b4bfd97f
Signed-off-by: Mitch Williams <mitch.a.williams@intel.com>
Signed-off-by: Jesse Brandeburg <jesse.brandeburg@intel.com>
Tested-by: Sibai Li <sibai.li@intel.com>
Signed-off-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit d15632d48629714439ecda3ea07322129e619e41)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40evf_main.c b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
index 1bb9deb..40a5e78 100644
--- a/drivers/net/ethernet/intel/i40evf/i40evf_main.c
+++ b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
@@ -2084,8 +2084,6 @@ static void i40evf_init_task(struct work_struct *work)
 
  netif_carrier_off(netdev);
 
- strcpy(netdev->name, "eth%d");
-
  adapter->vsi.id = adapter->vsi_res->vsi_id;
  adapter->vsi.seid = adapter->vsi_res->vsi_id; /* dummy */
  adapter->vsi.back = adapter;
-- 
1.7.1